components/eamxx/cime_config/namelist_defaults_scream.xml

```diff
@@ -213,6 +213,14 @@ be lost if SCREAM_HACK_XML is not enabled.
       <lambda_high type="real" doc="maximum value of stability correction.">0.04</lambda_high>
       <lambda_slope type="real" doc="slope of change from lambda_low to lambda_high.">2.65</lambda_slope>
       <lambda_thresh type="real" doc="stability threshold for which to apply more stability correction.">0.02</lambda_thresh>
+      <thl2tune type="real" doc="Temperature variance tuning factor">1.0</thl2tune>
+      <qw2tune type="real" doc="Moisture variance tuning factor">1.0</qw2tune>
+      <qwthl2tune type="real" doc="Temperature moisture covariance">1.0</qwthl2tune>
+      <w2tune type="real" doc="Vertical velocity variance">1.0</w2tune>
+      <length_fac type="real" doc="Length scale factor">0.5</length_fac>
+      <c_diag_3rd_mom type="real" doc="w3 factor">7.0</c_diag_3rd_mom>
+      <Ckh type="real" doc="Eddy diffusivity coefficient for heat">0.1</Ckh>
```

Comment:
`Ckh` and `Ckm` have the same documentation. Is this just a copy+paste that didn't get edited?

Reply:
yes, good catch.